src/main/java/io/github/balconyseats/vertx/helper/database/migration/ConfigDatabaseMigrationInitializationHandler.java

```diff
@@ -0,0 +1,96 @@
+package io.github.balconyseats.vertx.helper.database.migration;
+
+import io.github.balconyseats.vertx.helper.application.InitializationContext;
+import io.github.balconyseats.vertx.helper.application.configurer.InitializationHandler;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Configure and run migrations on database from configured migration type of {@link DatabaseMigration}.
```

Comment:
`Configure**s** and run**s**`. Since we are using third person singular form in other java docs.